Map.cpp

```diff
@@ -201,32 +214,47 @@ void Map::setText(Train *train, Station *station) {
         if(station->getType() == 1) {
```

Comment:
сделать enum для типов станций и switch вместо ифа? https://en.cppreference.com/w/cpp/language/enum
```suggestion
        switch(station->getType())
        {
             case StationType::LOVE :
```

Reply:
_Именно так_